install_config/install/docker_registry.adoc

```diff
@@ -947,15 +947,15 @@ auth:
 ====
 
 [[docker-registry-configuration-reference-middleware]]
-==== middleware
+==== Middleware
 
-The **repository** middleware extension allows to configure OpenShift middleware
-responsible for interaction with OpenShift and image proxying.
+The *repository* middleware extension allows to configure {product-title}
+middleware responsible for interaction with {product-title} and image proxying.
 
 [NOTE]
 ====
-The **middleware** section may not be overridden using environment variables.
-There are few exceptions though.
+The *middleware* section may not be overridden using environment variables.
+There are a few exceptions, however.
```

Comment:
This is probably a question for the original contributor, but it'd be a good idea to mention the specific exceptions.